content/intro/integrations/messaging/slack/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Slack"
+description: ""
+lead: ""
+date: 2022-08-15T02:48:57+00:00
+draft: false
+weight: 500
+toc: true
+tags:
+  - integrations
+  - messaging
+  - slack
+resources:
+  - src: code-risk-alerts-in-slack-via-blubracket.png
+---
+
+### Slack Integration
+
+{{< figure src="code-risk-alerts-in-slack-via-blubracket.png" caption="BluBracket integration with Slack can alert teams about code risks in new commits in real time." >}}
+
+### Requirements
```

Comment:
Should these pages be in http://docs.blubracket.com/how-to/integrations instead?